src/Tester/Tester.csproj

```diff
@@ -43,62 +43,67 @@
     <AssemblyOriginatorKeyFile>..\Orleans.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bond">
-      <HintPath>$(SolutionDir)packages\Bond.Runtime.CSharp.3.0.7\lib\net45\Bond.dll</HintPath>
```

Comment:
I did not touch that, that's nuget magic possibly. That is the problem is someone hand editing the csproj at ANY point that such things can happen. I did not see any guidance on this ;-) but IMHO noone should hand edit references and other stuff what can be overwritten by the IDE.

How to resolve?